README.md

```diff
@@ -27,6 +27,12 @@ script will remove that element from the record.
 }
 ```
 
+## Default Field Mapping File ##
+
+A default mapping file exists at `tests/artifacts/field_map.json` and should be
+used as a starting point to be compatible with most versions of the Pen Test Portal's
+findings JSON output.
```

Comment:
Please provide an example here in the README instead. That is our common practice as seen in [cisagov/cool-assessment-terraform](https://github.com/cisagov/cool-assessment-terraform#pre-requisites) for an appropriate Terraform variables file.

Reply:
> > > I would like the testing code added here removed. The newer lambda structure is based on creating a deployment and not a Python package so turning things into Python modules just for testing is not a workaround I can support.
> > 
> > 
> > Would you suggest something like the following, if @Yablargo wants to preserve the testing code?
> > ```
> > 1. Create a separate Python library based on the [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) skeleton that handles the heavy lifting.
> > 
> > 2. Add the mock and test code there so that the heavy lifting is tested.
> > 
> > 3. Leverage that Python library in this repo.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This does align with how I have done Lambda testing in the past, but I wanted to make sure you didn't have something else in mind.
> 
> Yes if the Python code is complicated enough it should be broken into its own Python project that is added as a dependency in the `Pipfile`s for the Lambda configuration. These Lambda projects should be designed around creating deployment artifacts and not storing a lot of logic. The only reason the logic remains in this project is because of timelines around getting this change out. Starting from scratch it is not how I would implement this functionality.

I guess this can serve my formal complaint on the matter, I think it is quite absurd to have the logic in this sort of project, and then claim that no testing is preferable to making a good faith attempt to improve testing because whoever wrote this should not have done so, years ago.

That said, I'll remove it if everyone agrees that we're better off with zero tests on things, although I'd suggest that we probably need a lot more across all of these projects since something as simple as package updates might break these sorts of things in interesting ways.

I guess I'd also say that something like the new lambda structure should have tests as part of the example and it well documented on how an end user is supposed to set up basic functional tests for these things. 

The golden rule **for me** has always been that if you touch functionality you should update tests to cover what you touched. I know that it is still sorely lacking, but the intent was to leave things in a slightly better place than I found them, and I try to use this as a guiding principle when working on any particular codebase.

@jsf9k I guess if you concur with above, I'll just drop all the test code and we can move on with our lives

Reply:
I'm gonna open a ticket for future work to add a comprehensive "defaults" to the docs, but I don't really know what that should look like right now and I think it is getting out of scope for this change. I've removed the original "default" text

Reply:
> @jsf9k I guess if you concur with above, I'll just drop all the test code and we can move on with our lives

I'd honestly be fine with leaving the test code in for now given that #57 exists, but since he is the architect of the Lambda skeleton I'll defer to @mcdonnnj.

I did add an acceptance criterion in #57 to remind us to revive the tests when that work takes place.

Reply:
> Please provide an example here in the README instead. That is our common practice as seen in [cisagov/cool-assessment-terraform](https://github.com/cisagov/cool-assessment-terraform#pre-requisites) for an appropriate Terraform variables file.

I just dropped it, this is not really related to the change, and really should be it's own separate PR.